apps/src/templates/teacherDashboard/TeacherDashboardNavigation.jsx

```diff
@@ -122,7 +122,7 @@ export default class TeacherDashboardNavigation extends Component {
 
   render() {
     const {listPosition, shouldScroll} = this.state;
-    if (this.props.showAITutorTab) {
+    if (this.props.showAITutorTab && teacherDashboardLinks.length <= 6) {
```

Comment:
- [ ] there might be a better way to do this, it was adding "AI Tutor" multiple times on re-render 😱 checking length fixes it for the common case, but links can also come in as props...? so this needs a little more investigation

Reply:
@lfryemason or @etaderhold I think you're most familiar with the Teacher Dashboard header and navigation at this point. Any advice for how to better address this? Thanks for considering!

Reply:
I think my preference would be to include the AI tutor tab in `teacherDashboardLinks` and filter it out if `showAITutorTab` is false. This would make sure we never add it accidentally multiple times and would also work if we add another tab later without needing to change this `6`.
Could you please put it in a `React.useEffect()` to make sure we don't filter on every render though?